common/src/main/java/com/redhat/cloud/notifications/openbridge/BridgeHelper.java

```diff
@@ -0,0 +1,107 @@
+package com.redhat.cloud.notifications.openbridge;
+
+import io.quarkus.cache.CacheResult;
+import org.eclipse.microprofile.config.inject.ConfigProperty;
+import org.eclipse.microprofile.rest.client.inject.RestClient;
+
+import javax.enterprise.inject.Produces;
+import javax.inject.Inject;
+import java.nio.charset.StandardCharsets;
+import java.util.Base64;
+import java.util.List;
+import java.util.Map;
+
+/**
+ *
+ */
+public class BridgeHelper {
+
+    @ConfigProperty(name = "ob.enabled", defaultValue = "false")
+    boolean obEnabled;
+
+    @ConfigProperty(name = "ob.kcUser")
+    String kcUser;
+    @ConfigProperty(name = "ob.kcPass")
+    String kcPass;
+    @ConfigProperty(name = "ob.bridge.name")
+    String ourBridge = "heiko-bridge";
+    @ConfigProperty(name = "ob.token.user")
+    String tokenUser;
+    @ConfigProperty(name = "ob.token.pass")
+    String tokenPass;
+
+    @Inject
+    @RestClient
+    BridgeApiService apiService;
+
+    @Inject
+    @RestClient
+    BridgeAuthService authService;
+
+
+    @Produces
+    public Bridge getBridgeIfNeeded() {
+
+        if (!obEnabled) {
+            return new Bridge("- OB not enabled -", "http://does.not.exist");
+        }
+
+        if (BridgeHolder.getInstance().getBridge() != null) {
+            return BridgeHolder.getInstance().getBridge();
+        }
+
+        String token = getAuthTokenInternal();
+
+        Map<String, Object> bridges = apiService.getBridges(token);
+        // List of bridges is in 'items'
+        List<Map<String, String>> bridgesList = (List<Map<String, String>>) bridges.get("items");
+
+        for (Map<String, String> b : bridgesList) {
+            if (b.get("name").equals(ourBridge)) {
```

Comment:
I assume `b.get("name")` will never be `null`. The test could be reverted otherwise:
```suggestion
            if (ourBridge.equals(b.get("name"))) {
```

Reply:
If this is null, we'll have bigger issues :-)
No friend of yoda-speak I am